client/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { createRoot } from 'react-dom/client';
+
+import App from './App';
+
+const root = createRoot(document.getElementById("app")!)
```

Comment:
Extra ! was added here, typo I'm guessing?

```suggestion
const root = createRoot(document.getElementById("app"))
```

Reply:
That was actually an intentional bit of syntax for a typescript problem:
![Screen Shot 2023-09-27 at 09 48 23](https://github.com/dyazdani/costume-shop-db/assets/99094815/07596efc-2f88-4b06-b9d0-e28261973062)